release/release_logs/1.11.0/microbenchmark.json

```diff
@@ -1,138 +1,126 @@
 {
-  "single_client_get_calls": [
-    24730.95712317265,
-    239.22426527698045
-  ],
-  "single_client_put_calls": [
-    45715.93513986856,
-    366.82602393524735
-  ],
-  "multi_client_put_calls": [
-    189007.7322366993,
-    747.0416719411013
-  ],
   "single_client_get_calls_Plasma_Store": [
-    5822.640574716487,
-    139.29057092881047
+    4417.483407539963,
```

Comment:
Is this a regression? cc @rkooo567 @scv119

Reply:
yup I think this is a regression.

Reply:
```
  "single_client_put_gigabytes": [
    9.203941431281354,
    7.382235276551273
  ],
```

This was 20GB in 1.10, so I think there was a regression in plasma store